proxy/api/protocol.go

```diff
@@ -28,6 +29,18 @@ type header struct {
 	flags  uint32
 }
 
+const maxPayloadLength = 1 * 1024 * 1024
```

Comment:
Except that this limit isn't the max length of a packet is it? just the length of the receive buffer. I'm hoping hyperstart can reconstruct bigger packets then the receive buffer size, but maybe that's too hopeful.